src/locale/__tests__/translations.test.ts

```diff
@@ -15,12 +15,12 @@ describe("translations", () => {
       };
     });
 
-    it("Should call add for every key", () => {
+    test("Should call add for every key", () => {
```

Comment:
I forgot if we were using it or test, so did a global search and these were the only ones using it, so changed them